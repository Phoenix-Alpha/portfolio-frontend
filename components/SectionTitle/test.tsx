import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SectionTitle } from '.'

describe('<SectionTitle />', () => {
  it('should renders correct <SectionTitle />', () => {
    const wrapper = shallow(<SectionTitle title="Section Title" />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
