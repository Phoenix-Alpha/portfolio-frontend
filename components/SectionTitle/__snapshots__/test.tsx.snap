// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTitle /> should renders correct <SectionTitle /> 1`] = `
<h3
  className="font-display text-xl lg:text-2xl pt-4 lg:pt-8 pb-4 lg:pb-8 font-bold text-white"
>
  Section Title
</h3>
`;
